src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -1245,9 +1245,11 @@ public void onReceive(Context context, Intent intent) {
                 if (linkedToRemotePath == null || isAscendant(linkedToRemotePath)) {
                     refreshListOfFilesFragment(true);
                 }
+                String localFilePath = intent.getStringExtra(Extras.EXTRA_FILE_PATH);
                 refreshSecondFragment(
                         intent.getAction(),
                         downloadedRemotePath,
+                        localFilePath,
```

Comment:
@svvashishtha I don't find the `localFilePath` variable to be used more then once in this function hence I suggest instead of creating a variable you could just do:-
`refreshSecondFragment(
                        intent.getAction(),
                        downloadedRemotePath,
                        intent.getStringExtra(Extras.EXTRA_FILE_PATH),
                        intent.getBooleanExtra(Extras.EXTRA_DOWNLOAD_RESULT, false)
                );`